src/main/java/org/cascadebot/cascadebot/commands/fun/ColorCommand.java

```diff
@@ -6,99 +6,23 @@
 
 package org.cascadebot.cascadebot.commands.fun;
 
-import net.dv8tion.jda.api.EmbedBuilder;
 import net.dv8tion.jda.api.entities.Member;
-import org.cascadebot.cascadebot.commandmeta.Argument;
-import org.cascadebot.cascadebot.commandmeta.ArgumentType;
 import org.cascadebot.cascadebot.commandmeta.CommandContext;
 import org.cascadebot.cascadebot.commandmeta.ICommandMain;
 import org.cascadebot.cascadebot.commandmeta.Module;
-import org.cascadebot.cascadebot.messaging.MessageType;
-import org.cascadebot.cascadebot.messaging.MessagingObjects;
 import org.cascadebot.cascadebot.permissions.CascadePermission;
-
-import java.awt.Color;
-import java.util.Set;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
+import org.cascadebot.cascadebot.utils.ColorUtils;
 
 public class ColorCommand implements ICommandMain {
 
-    private static final Pattern HEX_COLOR = Pattern.compile("#([A-Fa-f0-9]+)");
-    private static final Pattern DECIMAL_COLOR = Pattern.compile("([0-9]{1,8})");
-    private static final Pattern RGB_COLOR = Pattern.compile("(\\d{1,3}),(\\d{1,3}),(\\d{1,3})");
-    private static final Pattern BINARY_COLOR = Pattern.compile("([0-1]+)");
-
-    private String getHex(int r, int g, int b) {
-        return String.format("%02x%02x%02x", r, g, b);
-    }
-
     @Override
     public void onCommand(Member sender, CommandContext context) {
         if (context.getArgs().length != 1) {
             context.getUIMessaging().replyUsage(this);
```

Comment:
This is that and thus lettuce

Reply:
?

Reply:
Sorry, it says replyUsage(this)

Reply:
Can you please, for crying out loud, include what is wrong?

Reply:
> Sorry, it says replyUsage(this)
It should be `replyUsage()`

Reply:
(Next time)
You left me wondering what the hell you were talking about xD

Reply:
By the way, don't blame me for that @JamesHawkinss . It was done by binary